src/pages/Naming.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react'
+import Dropdown from '../components/pulldown/DropDown'
+const naming: React.FC = () => {
```

Comment:
基本的にコンポーネントは アッパーキャメルケースで書かれることが多いよ
あと、何ページ作る想定かわからんけど、作成後の一覧ページとかあるなら NamingCretePageとした方がいい気がする
それと default export だと使う側で import 名を変更できるから named export の方がいいよ（この違いは調べておくれ）

```suggestion
export const NamingCreatePage: React.FC = () => {
```

Reply:
ページ数で言ったら１ページで完結させる予定の場合は変更なくてもいいんかな？？